models/garden.js

```diff
@@ -11,6 +11,8 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
+      Garden.hasMany(models.GardenPlant);
+      Garden.belongsToMany(models.Plant, { through: models.GardenPlant });
```

Comment:
Here we see the data model ASSOCIATIONS defined between Garden/GardenPlant/Plant.